src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -752,11 +752,34 @@ def assign_college_simulants_guaridans(self, pop: pd.DataFrame) -> pd.DataFrame:
                     households_guardian_types[guardian_type]["race_ethnicity"] == race,
                     "reference_person_id",
                 ]
-                race_guardian_ids = self.randomness.choice(
-                    race_college_sims_with_guardian_idx,
-                    choices=race_reference_person_ids,
-                    additional_key=f"{race}_{guardian_type}_guardian_ids",
-                )
+                if (
+                    race_reference_person_ids.empty & len(race_college_sims_with_guardian_idx)
```

Comment:
use `and` here for boolean operator

Reply:
I've changed it but does this make a difference functionally?

Reply:
Full disclosure - I still struggle with when to use bitwise operations (&). For this particular case, no, the "answer" is the same. But it's just not really the correct approach since you're dealing with boolean logic here.